config/locales/fr.yml

```diff
@@ -0,0 +1,76 @@
+fr:
+  temporale:
+    solemnity:
+      nativity: La Nativité du Seigneur
+      holy_family: La Sainte Famille de Jésus, Marie et Joseph
+      mother_of_god: Octave de Noël, de Marie, Mère de Dieu
+      epiphany: L'Epiphanie du Seigneur
+      baptism_of_lord: Le Baptême du Seigneur
+      ash_wednesday: Mercredi des Cendres
+      palm_sunday: Dimanche des Rameaux de la Passion du Seigneur
+      good_friday: Vendredi de la Passion du Seigneur
+      holy_saturday: Samedi Saint
+      easter_sunday: Dimanche de Pâques de la Résurrection du Seigneur
+      ascension: Ascension du Seigneur
+      pentecost: Dimanche de Pentecôte
+      holy_trinity: La Très Sainte Trinité
+      corpus_christi: Le Très Saint Corps et Sang du Christ
+      sacred_heart: Le Sacré-Coeur de Jésus
+      immaculate_heart: Le Coeur Immaculé de Marie
+      christ_king: Notre Seigneur Jésus Christ, Roi de l'Univers
+    ordinary:
+      sunday: '%{week} Dimanche de Temps Ordinaire'
+      ferial: '%{weekday}, %{week} semaine de Temps Ordinaire'
+    advent:
+      sunday: '%{week} Dimanche de l''Avent'
+      ferial: '%{weekday}, %{week} semaine de L''Avent'
+    christmas:
+      nativity_octave:
+        ferial: 'Jour %{day} de l''Octave de Noël'
+      sunday: '%{week} Dimanche après la Naissance du Seigneur'
+      ferial: '%{weekday} après l''Octave de Noël'
+      after_epiphany:
+        ferial: '%{weekday} après l''Epiphanie'
+    lent:
+      after_ashes:
+        ferial: '%{weekday} après le Mercredi des Cendres'
+      sunday: '%{week} Dimanche de Carême'
+      ferial: '%{weekday}, %{week} semaine de Carême'
+      holy_week:
+        ferial: '%{weekday} de la Semaine Sainte'
+    easter:
+      octave:
+        ferial: 'Pâques %{weekday}'
+      sunday: '%{week} Dimanche de Pâques'
+      ferial: '%{weekday}, %{week} semaine de Pâques'
+  rank:
+    '1_1': Triduum Pascal
+    '1_2': Jours liturgiques primaires
+    '1_3': Solennités du Calendrier Général
+    '1_4': Solennités appropriées
+    '2_5': Fête du Seigneur du Calendrier Général
+    '2_6': Dimanche désavantagés
```

Comment:
While my French skills are really poor, I'm slightly suspicious concerning the translation of "Unprivileged Sundays" as "Dimanche désadvantagés". The full paragraph in the Table of Liturgical Days which I shortenned to "Unprivileged Sundays" reads "Sundays of the Christmas season and Sundays in Ordinary Time." The point is, these Sundays are in no way "disadvantaged", they are just "simple Sundays", without an additional privileged status. Most of the Sundays during the liturgical year belong to this category. Is this what the French adjective "désadvantagé" means, or is there a better word to express the idea?